src/Mutex/WorkflowWithMutexInput.cs

```diff
@@ -0,0 +1,3 @@
+namespace TemporalioSamples.Mutex;
+
+public record WorkflowWithMutexInput(string ResourceId, int SleepForMs = 500, int LockTimeoutMs = 1000);
```

Comment:
Feel free to accept `TimeSpan`s here if you are willing to pass them around for other use cases